openmdao/units/units.py

```diff
@@ -880,5 +880,6 @@ def get_conversion_tuple(src_units, target_units):
 
 
 # Load in the default unit library
-with open(os.path.join(os.path.dirname(__file__), 'unit_library.ini')) as default_lib:
+import pkg_resources
+with pkg_resources.resource_stream(__name__, "unit_library.ini") as default_lib:
     import_library(default_lib)
```

Comment:
Python 3 doesn't seem to like this:

```File "/Users/jwilliams/projects/openmdao-updates/python311/OpenMDAO/mpitest/test_brute_force.py", line 10, in <module>
    from openmdao.api import Problem, Group, ParallelGroup, \
  File "/Users/jwilliams/projects/openmdao-updates/python311/OpenMDAO/openmdao/api.py", line 2, in <module>
    from openmdao.components.constraint import ConstraintComp
  File "/Users/jwilliams/projects/openmdao-updates/python311/OpenMDAO/openmdao/components/constraint.py", line 5, in <module>
    from openmdao.components.exec_comp import ExecComp
  File "/Users/jwilliams/projects/openmdao-updates/python311/OpenMDAO/openmdao/components/exec_comp.py", line 12, in <module>
    from openmdao.core.component import Component
  File "/Users/jwilliams/projects/openmdao-updates/python311/OpenMDAO/openmdao/core/component.py", line 16, in <module>
    from openmdao.core.system import System
  File "/Users/jwilliams/projects/openmdao-updates/python311/OpenMDAO/openmdao/core/system.py", line 19, in <module>
    from openmdao.units.units import get_conversion_tuple
  File "/Users/jwilliams/projects/openmdao-updates/python311/OpenMDAO/openmdao/units/units.py", line 885, in <module>
    import_library(default_lib)
  File "/Users/jwilliams/projects/openmdao-updates/python311/OpenMDAO/openmdao/units/units.py", line 729, in import_library
    _UNIT_LIB.readfp(libfilepointer)
  File "/Users/jwilliams/.pyenv/versions/3.11.1/lib/python3.11/configparser.py", line 778, in readfp
    self.read_file(fp, source=filename)
  File "/Users/jwilliams/.pyenv/versions/3.11.1/lib/python3.11/configparser.py", line 733, in read_file
    self._read(f, source)
  File "/Users/jwilliams/.pyenv/versions/3.11.1/lib/python3.11/configparser.py", line 1051, in _read
    if line.strip().startswith(prefix):
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
TypeError: startswith first arg must be bytes or a tuple of bytes, not str
```

Should probably make sure something similar isn't happening in the other places where you're using `resource_stream` as well.